pages/learn/manage/device-statuses.md

```diff
@@ -11,36 +11,69 @@ Device statuses are displayed on the Devices page and the Device Summary page. A
 |----------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **Online**                                   | The device is online and is communicating with the cloud within the correct interval. In this state, the device is connected to cloudlink and has had recent API communications. |
-| **Online (VPN&#160;Only)**<sup>1</sup>       | The device is online, is connected to cloudlink, but has **no recent API communications**.                                                                                       |
-| **Online (Heartbeat&#160;Only)**<sup>2</sup> | The device is online, has recent API communications, but is **not connected to cloudlink**.                                                                                      |
-| **Offline**                                  | The device is offline.                                                                                                                                                           |
-| **Configuring**                              | The device is applying OS configuration.                                                                                                                                         |
-| **Updating**                                 | The device is updating to a new release.                                                                                                                                         |
+| **Operational**                              | The device is functioning normally and communicating with the cloud within the expected interval. In the standard configuration, this means both [cloudlink][cloudlink] and the device heartbeat API are working. If [cloudlink is disabled][bandwidth-reduction], the device will still appear `Operational` if the heartbeat API is functional.   |
+| **Reduced Functionality**<sup>1</sup>        | The device is connected to the backend but is experiencing limited functionality due to either **missing recent Heartbeat API communications** or **lack of connection to cloudlink**.                                                                                       |
+| **Disconnected**                                  | The device is not connected to cloudlink and has not communicated via the Heartbeat API for a duration exceeding the [API polling interval][poll-interval]. This status indicates that the device is completely offline and unable to send or receive any data from the cloud.                                                                                                                                                           |
```

Comment:
You've linked to [this page](https://d89e6939.balenacloud-docs.pages.dev/learn/manage/configuration/#variable-list) here but from a UX perspective, the API polling interval or poll interval isn't mentioned here? It's confusing from UX